requirements.txt

```diff
@@ -40,7 +40,7 @@ lazy-object-proxy==1.3.1
 linecache2==1.0.0
 lz4==2.0.0
 Markdown==2.6.11
-MarkupSafe==1.0
+MarkupSafe==1.1.1
```

Comment:
This is a transitive dependency of something but I'm not sure what.

Changelog: https://markupsafe.palletsprojects.com/en/1.1.x/changes/